Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM ubuntu:18.04
 
 RUN apt update
 RUN DEBIAN_FRONTEND=noninteractive apt install -yq --no-install-recommends php php-json php-curl php-ctype php-iconv php-pdo php-mysql php-dom php-zip composer apache2 libapache2-mod-php mysql-client sudo
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends cron
```

Comment:
@daria-sieroshtan 
Why did you remove cron?

Reply:
It doesn't install properly from dockerfile